EssayAnalyzer.Api/Brokers/Storages/StorageBroker.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+
+namespace EssayAnalyzer.Api.Brokers.Storages
+{
+    public partial class StorageBroker : EFxceptions.EFxceptionsContext, IStorageBroker
+    {
```

Comment:
Fix the name spaces of `EFxceptions.EFxceptionsContext` ex: `using Microsoft.EntityFrameworkCore;`

Reply:
namespaces fixed